TODO.md

```diff
@@ -0,0 +1,6 @@
+# TODO
```

Comment:
I will yeet this file prior to merging, it was more just to remind me to discuss the items in the file